### PR TITLE
vendor: github.com/moby/buildkit 65f4948

### PR DESCRIPTION
**Pull request by crazy-max**
https://github.com/moby/buildkit/compare/d7744bcb3532...65f494851ae3

also updates containerd with changes from `release/1.6` branch that fixes a deadlock: https://github.com/containerd/containerd/compare/v1.6.2...5ff8fce1fcc6

Signed-off-by: CrazyMax <crazy-max@users.noreply.github.com>

### CODE REVIEW

**Comment by crazy-max** [comment]
keeping in draft for now as there are dependencies we might want to vendor independently. cc @thaJeztah

File: vendor.mod [hunk 1]
```diff
@@ -61,7 +61,7 @@ require (
 	github.com/moby/term v0.0.0-20210619224110-3f7ff695adc6
 	github.com/morikuni/aec v1.0.0
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/opencontainers/image-spec v1.0.2-0.20211117181255-693428a734f5
+	github.com/opencontainers/image-spec v1.0.3-0.20211202183452-c5a74bcca799
```
Comment (thaJeztah) [inline vendor.mod:64]: Actually; let me update this one there as well, to align with containerd

File: vendor.mod [hunk 2]
```diff
@@ -10,16 +10,16 @@ require (
...
 	github.com/Microsoft/hcsshim v0.9.2
 	github.com/RackSec/srslog v0.0.0-20180709174129-a4725f04ec91
 	github.com/armon/go-radix v0.0.0-20180808171621-7fddfc383310
 	github.com/aws/aws-sdk-go v1.31.6
 	github.com/bsphere/le_go v0.0.0-20170215134836-7a984a84b549
 	github.com/cloudflare/cfssl v0.0.0-20180323000720-5d63dbd981b5
 	github.com/containerd/cgroups v1.0.3
-	github.com/containerd/containerd v1.6.2
-	github.com/containerd/continuity v0.2.2
+	github.com/containerd/containerd v1.6.3-0.20220401172941-5ff8fce1fcc6 // release/1.6 branch
```
[the remark this answers is not in the document]
Comment (crazy-max) [reply vendor.mod:21]: Great thanks. Done to on BuildKit repo https://github.com/moby/buildkit/pull/2833. Will update vendoring when merged.

**Comment by thaJeztah** [comment]
Forgot that this one also had the Version changes, so reopening (needs a rebase though 😅)

**Pull request closed without merging**